pmda/test/test_parallel.py

```diff
@@ -111,3 +111,16 @@ def test_nblocks(analysis, n_blocks):
 def test_guess_nblocks(analysis):
     analysis.run(n_jobs=-1)
     assert len(analysis._results) == joblib.cpu_count()
+
+def test_attrlock():
```

Comment:
pylint is pretty picky – it wants two blank lines here https://travis-ci.org/MDAnalysis/pmda/jobs/403217647#L2147

Reply:
this might rather be the pep8 linter.